server.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-undef */
+const express = require('express');
```

Comment:
Are you using express instead of live server?

Reply:
Yes, I am using express is okay?

Reply:
Yeah its ok